README.dist.md

```diff
@@ -1,46 +1,71 @@
 # WordPress Site Starter
-To use this, clone the WordPress Site Starter into the new project repo. Update this to have info about things to note how to be successful using the starter.
+This is the README for your new site. Feel free to update any of this info to match your project. 
+
+## Links
+-   [Production](#UPDATETHIS)
+-   [Staging](#UPDATETHIS)
+-   [Development](#UPDATETHIS)
 
 ## Requirements
 * [Composer](https://getcomposer.org/) - [Installation](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)
 * [DDEV](https://ddev.readthedocs.io/en/stable/) - [Installation](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/)
-* [Docker](https://docs.docker.com/desktop/install/mac-install/) (or compatible container alternative)
+* [Docker](https://docs.docker.com/desktop/install/mac-install/) (or compatible container alternative like  OrbStack)
 * For ACF Pro, create an `auth.json` file in `wp-content/mu-plugins/viget-base/` from the [ACF Website](https://www.advancedcustomfields.com/my-account/view-licenses/). (Credentials are in 1Password)
 
+
 ## Setup and Running
 
-Download and install WordPress core files if this is the first time running DDEV
-`ddev wp core download`
+Run `ddev start` to start the site. This will install WordPress files, composer packages, npm packages, and start the DDEV server. Once the server is started DDEV will automatically start Vite for local development. 
+
+If this is the first time this project has been set up: 
+1. Select the desired language.
+2. Fill out the site information.
+3. Then click "Install WordPress"
+4. Once WordPress has been set up login with your user information.
 
-Start local developement server
-`bin/start`
+If this is an existing project, get an database download from production, staging or another team member. Steps to download a database are different depending on the hosting server. 
+
+Once you have the database file import the database into DDEV using:
+```bash
+ddev import-db --file=FILENAME.sql.gz
+```
+Then login using the username/password for the user on that database.
```

Comment:
Actually, there's a step in the `post-install` hook that asks for the database source, and they can provide the path to the file to be imported, which will occur when a dev is setting up a project for the first time.